api_app/services/cost_service.py

```diff
@@ -20,14 +23,16 @@
 class ResultColumnDaily(Enum):
     Cost = 0
     Date = 1
-    Tag = 2
-    Currency = 3
+    ResourceGroup = 2
```

Comment:
probably ok. but it is ok to add a number in the middle? backward compatibility wise

Reply:
Thanks Elad, those enums are used merely to maps the column positions in the Azure cost management API results, instead of working with hard coded numeric positions. Adding the RresourceGroup group-by column to the result in the query definition resulted in this uncontrollable order.